libheif/plugins/encoder_svt.cc

```diff
@@ -712,20 +718,28 @@ struct heif_error svt_encode_image(void* encoder_raw, const struct heif_image* i
 #endif
 
 
+#if !SVT_AV1_CHECK_VERSION(3, 0, 0)
```

Comment:
Is there a corresponding flag in v3.0.0 or was this simply removed?

Reply:
It was removed in: https://gitlab.com/AOMediaCodec/SVT-AV1/-/commit/92d44bdcc6331ecc28338f99207be912bfd05729#900962ec0dfb11881a5f25ce6fcad8e815c8fd45_341_333

libavif also removed it completely: https://github.com/AOMediaCodec/libavif/commit/e649830525d37f93e83a28b2a7dc4afbf0eb12d0#diff-29748a2db41273018a16b71ccb60bcd7b632b86c78cc69bb88784702122ebddeL127-L132

Reply:
Thank you for the background information.